### ISSUE TITLE
bug:  "find file" icon not shown in MacOS

### ISSUE THREAD

**Issue by ChenZhongPu** [LazyVim/LazyVim#456]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the exsiting issues of plugins related to this issue

### Neovim version (nvim -v)

0.8.3

### Operating system/version

MacOS 13.2

### Describe the bug

The "find file" icon is not shown.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/2310958/226103149-f5b03fe5-b9f1-4feb-9d99-537fe4436139.png">

This is caused by `ui.lua` using [alpha-nvim](https://github.com/goolord/alpha-nvim):

```lua
dashboard.button("f", " " .. " Find file", ":Telescope find_files <CR>"),
```

It seems that icon `\uF002` is missing in MacOS (I am using Iterm2), while it works in my Linux machine. I tried both `hack-nerd-font` and `MesloGS NF`.

### Steps To Reproduce

Type `nvim`, or `echo -e "\uF002"`.

### Expected Behavior

The "find file" icon should be shown.

**Comment by ChenZhongPu** [comment]
I found it may caused by `iTerm2`. I tested in `Terminal`, instead of `iterm2`, and `\uF002` is shown correctly.

Note that I am using the same font *MesloGS NF*.

**Comment by folke** [comment]
You need to use a proper terminal and make sure you have an up to date Nerd Font as mentioned in the requirements

**Comment by ChenZhongPu** [comment]
As I mentioned, even if I use Nerd Font (font-hack-nerd-font: 2.3.3) in iTerm2, `\uF002` cannot be displayed correctly.

<img width="593" alt="image" src="https://user-images.githubusercontent.com/2310958/226150750-fecf7899-c068-4ca7-88f2-1b3c76f27766.png">

I think it may have something to do with my iTerm2 settings.

**Comment by loqusion** [comment]
I've reproduced this issue in a fresh MacOS virtual machine with minimal installation. Here are the steps I took:

1. Install iTerm2 (official website)
2. Install a nerd font (official website) (I tested multiple)
3. Select the nerd font in iTerm2 settings
4. Install homebrew
5. `brew install neovim`
6. `git clone https://github.com/LazyVim/starter ~/.config/nvim`
7. `nvim`

<img width="1407" alt="Screenshot 2023-04-19 at 3 44 11 PM" src="https://user-images.githubusercontent.com/38332081/233195730-d16166c9-dfc4-4b07-a6ef-1923dc60d348.png">

Given this, I doubt it's an issue with iTerm2 settings (unless there's something wrong with the default settings). I could open an issue on the iTerm2 repository, but given that there's currently nearly 3,000 open issues I doubt it'll receive much attention.

Is there anything we can do about this?

**Comment by folke** [comment]
Still the same, terminal issue or nerdf fonts issue....

**Comment by ChenZhongPu** [comment]
> I've reproduced this issue in a fresh MacOS virtual machine with minimal installation. Here are the steps I took:
> 
>     1. Install iTerm2 (official website)
> 
>     2. Install a nerd font (official website) (I tested multiple)
> 
>     3. Select the nerd font in iTerm2 settings
> 
>     4. Install homebrew
> 
>     5. `brew install neovim`
> 
>     6. `git clone https://github.com/LazyVim/starter ~/.config/nvim`
> 
>     7. `nvim`
> 
> 
> <img alt="Screenshot 2023-04-19 at 3 44 11 PM" width="1407" src="https://user-images.githubusercontent.com/38332081/233195730-d16166c9-dfc4-4b07-a6ef-1923dc60d348.png">
> 
> Given this, I doubt it's an issue with iTerm2 settings (unless there's something wrong with the default settings). I could open an issue on the iTerm2 repository, but given that there's currently nearly 3,000 open issues I doubt it'll receive much attention.
> 
> Is there anything we can do about this?

I have switched to `kitty` from iTerm2, and now it works.

**Comment by droptopx** [comment]
For future reference: This exact same issue (along with other codepoints that don't print correctly) was also talked about in https://github.com/ryanoasis/nerd-fonts/issues/1149. There, a comment refers to https://gitlab.com/gnachman/iterm2/-/issues/10431#note_1005459893, which indicates that this issue has been fixed in nightly and the fix will be shipped with iTerm 3.5.x.

**Comment by NatureLR** [comment]
the same problem
